### DISCUSSION TITLE
I have a problem with verification of my email address and I can't find something to fix it

### DISCUSSION THREAD

**Discussion by cyberpion-dev** [community/community#108232] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![Screenshot_20240214-014613](https://github.com/community/community/assets/118452356/93d0bc96-f90a-460d-8af4-f974e99c4e9f)

**Comment by VerzatileDev** [comment]
Might be a good idea to read the following documentation in relation to the Email issue: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/verifying-your-email-address  

It might be due to a combination of different areas:

- Same browser for both actions Verifying on Email and then seeing the Github end
- Verification has expired before being able to be processed
- It might be an unsupported address (such for Temporary emails as an example).
- It might be entirely an issue in relation to the github app, I would recommend try it again using the browser on your phone and sign in from there or sign out of the app and try again.

- It could as well be that the validation from the server is miss matching from the actual verification, and it might take time. 
- Make sure this is your primary Email address, otherwise in some instances the other email if you have one may be blocking the addition of a new one.

**Reply by cyberpion-dev** [reply]
I got it, thak u so much. I'm used to confirming via phone. That was a problem 😅

**Comment by Lanrad23** [comment]
It happened to me a while ago on another website. Well, I registered from my phone and didn't even notice that on my email, I put .con instead of .com. Surprisingly, it was challenging to register another account with the correct email because the name and phone number were the same.
